cogs5e/homebrew.py

```diff
@@ -85,9 +85,11 @@ async def bestiary_delete(self, ctx, *, name):
 
     @bestiary.command(name="import")
     async def bestiary_import(self, ctx, url):
-        """
-        Imports a bestiary from [CritterDB](https://critterdb.com/).
+        """Imports a bestiary from [Bestiary Builder](https://bestiary.stevnbak.dk) or [CritterDB](https://critterdb.com/).
```

Comment:
Should this be bestiarybuilder.com, like the code below?

Reply:
Good catch, fixed!